.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Alpine",
-  "image": "mcr.microsoft.com/vscode/devcontainers/base:0-alpine-3.13",
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:0-alpine-3.16",
```

Comment:
I'd suggest to go to Alpine 3.20 directly, if that is possible.

Reply:
Thanks, I have change to the Alpine 3.20.